### PR TITLE
Fix build cache issues

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -116,13 +116,12 @@ jobs:
 
 cache:
   directories:
-    - "$HOME/.cache/coursier"
+    - "$HOME/.coursier/cache"
```
Comment (mkurz) [inline .travis.yml]: So you want to change that back @marcospereira? Please have a look at the discussion I had with @dwijnand last week: https://github.com/playframework/playframework/pull/9967/files#r368914243

Comment (marcospereira) [reply .travis.yml]: Hum, I see it here now:

https://get-coursier.io/docs/cache.html#location

Thanks, Matthias.

**Comment by mkurz** [comment]
I am pretty sure we can backport to 2.8.x because also in 2.8.x we now use sbt-dynver.

**Comment by mkurz** [comment]
@Mergifyio backport 2.8.x

**Pull request merged**